src/process.c

```diff
@@ -487,18 +531,22 @@ cc_oci_vm_netcfg_get (struct cc_oci_config *config,
  * \param shim_args_fd Writable file descriptor caller should use to
  *   send the proxy_socket_fd to the child before \ref CC_OCI_SHIM is
  *   launched.
+ * \param shim_socket_fd Writable file descriptor caller should use to
+ *   send the proxy IO fd to child before \ref CC_OCI_SHIM is launched.
  *
  * \return \c true on success, else \c false.
  */
 static gboolean
 cc_shim_launch (struct cc_oci_config *config,
 		int *child_err_fd,
-		int *shim_args_fd)
+		int *shim_args_fd,
+		int *shim_socket_fd)
```

Comment:
Need to add `shim_socket_fd` to the args test below which currently checks `child_err_fd` and `shim_args_fd`.